asn1tools/codecs/uper.py

```diff
@@ -1044,6 +1058,14 @@ def compile_type(self, name, type_descriptor, module_name):
 
         return compiled
 
+    def get_extension(self, values):
+        extension = None
+        
+        for value in values:
+            if value == '...':
+                extension = []
+        return extension
```

Comment:
Does this function return anything but `None`? I mean, the loop iterates over the keys, which are integers. Comparing an integer to `'...'` will never be true. Or am I missing something?

Reply:
You are right, I fixed it